thumbor/engines/__init__.py

```diff
@@ -368,5 +368,17 @@ def strip_icc(self):
     def extract_cover(self):
         raise NotImplementedError()
 
+    def has_transparency(self):
+        raise NotImplementedError()
```

Comment:
Do you think it might be considered BC break? Since engines must implement this method to avoid exception in runtime.